sdk/applicationinsights/microsoft-azure-applicationinsights-query/pom.xml

```
@@ -14,7 +14,7 @@
     <name>Microsoft Azure SDK for Application Insights Query API</name>
     <description>
         Please note, a newer package azure-monitor-query for querying logs from Azure Monitor is available at
-        https://search.maven.org/artifact/com.azure/azure-monitor-query as of October 2021. Read the migration guide
+        https://central.sonatype.com/search?namespace=com.azure&amp;name=azure-monitor-query as of October 2021. Read the migration guide
```

Since this is in pom, this doesn't get hyperlinked and the link should be copy/paste-able.

```suggestion
        https://central.sonatype.com/search?namespace=com.azure&name=azure-monitor-query as of October 2021. Read the migration guide
```

---

If we switch to the overview link instead, we don't need to use the query params at all.